examples/neural_networks/plot_elm_hyperparameters.py

```diff
@@ -0,0 +1,80 @@
+"""
+===========================================================
+Extreme Learning Machines: Effect of tuning hyperparameters
+===========================================================
+
+This example first demonstrates how sklearn.grid_search.GridSearchCV can
+scan through the space of 2 significant hyperparameters C and weight_scale,
```

Comment:
I would use "important" not "significant" her and in the next sentence. "significant" has a very specific meaning in stats. There is not really a danger of confusion here, still I feel "significant" a bit odd.